Sources/MapboxCoreNavigation/Distance/LineDistanceInfo.swift

```diff
@@ -12,18 +12,27 @@ public struct LineDistanceInfo {
      */
     public let distanceToEntry: CLLocationDistance
 
-    /** Distance along the road graph from current position to exit point of object in meters. */
+    /** Distance along the road graph from current position to the most likely exit point in meters. */
     public let distanceToExit: CLLocationDistance
 
-    /** Distance along the road graph from current position to end of road object */
+    /** Distance along the road graph from current position to end of the most distant exit */
     public let distanceToEnd: CLLocationDistance
 
-    /** If we enter road object from it's start, if already "within" object - always false */
+    /** If we enter road object from its start, if already "within" object - always false */
     public let isEntryFromStart: Bool
 
-    /** Length of "long" objects */
+    /** Length of objects, measured in meters. */
```

Comment:
```suggestion
    /** Length of the road object, measured in meters. */
```

(Also below in the initializer’s documentation comment.)